### PR TITLE
Adjust order of expected/actual in how to file a bug

### PR DESCRIPTION
**Pull request by petele**
Changes proposed in this pull request:
- Adjusts order of 'expected' and 'actual' to match the Chromium bug template

### CODE REVIEW

File: src/site/content/en/blog/how-to-file-a-good-bug/index.md [hunk 1]
```diff
@@ -152,20 +152,20 @@ What steps will reproduce the problem?
...
 0. INSTALL: Available (logged when `beforeinstallprompt` event fired)
 1. INSTALL_PROMPT_RESPONSE: {outcome: "accepted", platform: "web"}
    (logged when beforeinstallprompt.prompt()` resolves)
 2. INSTALL: Success (logged when `appinstalled` event fired)
+
+What is the actual result? In the console:
+0. INSTALL: Available (logged when `beforeinstallprompt` event fired)
+1. INSTALL: Success (logged when `appinstalled` event fired)
+2. INSTALL_PROMPT_RESPONSE: {outcome: "accepted", platform: "web"}
+   (logged when beforeinstallprompt.prompt()` resolves)
```
Comment (jpmedley) [inline src/site/content/en/blog/how-to-file-a-good-bug/index.md:168]: I don't understand what I'm looking at, so I don't know how to review it. Is this a procedure? A list of possible outputs from something?

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Comment by robdodson** [comment]
@petele any thoughts on joe's comment?

**Comment by samthor** [comment]
I don't think this needs content review, it's already been reviewed. This is just rearranging "expected" before "actual", to match most testing environments.

**Review by samthor — APPROVED** [review]

**Comment by jpmedley** [comment]
@samthor I don't know who your content reviewer was, but it's possible he or she suffered from the "curse of knowledge". It may have passed because the reviewer knew what that section described before the reviewed the article. As someone who's never seen it before, I can't work it out.

**Comment by jpmedley** [comment]
@petele @MichaelSolati This _really_ fell through the cracks. Do you want me to open a new PR?

File: src/site/content/en/blog/how-to-file-a-good-bug/index.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ authors:
   - robertnyman
   - petelepage
 date: 2020-06-15
-updated: 2020-06-15
+updated: 2021-05-02
```
Comment (jpmedley) [inline src/site/content/en/blog/how-to-file-a-good-bug/index.md:11]:
```suggestion
updated: 2022-02-22
```

**Review by jpmedley — CHANGES_REQUESTED** [review]
My 'suggestion' is to update the revision date. Other than that, this is good.

File: src/site/content/en/blog/how-to-file-a-good-bug/index.md [hunk 3]
```diff
@@ -98,7 +98,7 @@ bug getting fixed.
 Here are a few tips for minimizing a test case:
 
 * Download the web page, add
-  [`<base href="https://original.url">`](https://developer.mozilla.org/docs/Web/HTML/Element/base)
+  [`<base href="https://original.url">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base)
```
Comment (rachelandrew) [inline src/site/content/en/blog/how-to-file-a-good-bug/index.md:101]:
```suggestion
  [`<base href="https://original.url">`](https://developer.mozilla.org/docs/Web/HTML/Element/base)
```

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**